recipe/meta.yaml

```diff
@@ -1,23 +1,29 @@
 {% set name = "fonttools" %}
-{% set version = "3.24.2" %}
-{% set sha256 = "ff8caa3fe923984147e4c007f550ce96b10ead1c6d8085a97465c8c3d6ad2a04" %}
+{% set version = "3.28.0" %}
+{% set sha256 = "ebf2ee25a6060e8551880a3b05d6ecadcdcfbd1b32fc272ff7b3acdb22945b6f" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
```

Comment:
Why the move to pypi?

Reply:
PyPI publishes the sha256 for files and has some guarantees about the immutability of the files.  GitHub archives can and do change and I know of no manner of finding the hash besides downloading the file.

Reply:
Could we use this? I think the releases downloads are stable
https://github.com/fonttools/fonttools/releases/download/3.28.0/fonttools-3.28.0.zip

Reply:
Thank you!